### PR TITLE
FIX: inputMapStyle Kepler to load the defaultMapBoxStyleUrl

### PR DESCRIPTION
**Pull request by OlivPi**

### CODE REVIEW

File: src/components/map/Map.js [hunk 1]
```diff
@@ -44,7 +43,7 @@ export default function MapContainer() {
               mapboxApiAccessToken={mapboxToken}
               width={width}
               height={height}
-              appName="Datatlas"
```
Comment (lutangar) [inline src/components/map/Map.js:47]: Should be left intact or be configured via instance configuration.

**Review by lutangar — CHANGES_REQUESTED** [review]

**Comment by lutangar** [comment]
@OlivPi could you rebase on `dev` please?

File: src/styles/styled.d.ts [hunk 2]
```diff
@@ -50,5 +50,17 @@ declare module 'styled-components' {
       backgroundColor: string;
       borderRadius: string;
     };
+
+    // Header: {
```
Comment (lutangar) [inline src/styles/styled.d.ts:54]: You probably should remove this commented code.

**Review by OlivPi — COMMENTED** [review]
I did a little cleaning (props and comment) on button's components and modal.

There is one comment left on ButtonSwitch style... Because i'm waiting some instruction about its design.

Sorry for the multi commits on the same review.

**Review by lutangar — APPROVED** [review]

**Comment by lutangar** [comment]
Superseded by #133

**Pull request closed without merging**